frontend_v2/src/app/components/challenge/challengesubmit/challengesubmit.component.html

```diff
@@ -236,7 +236,8 @@ <h5 class="fw-light">Make Submission</h5>
                     </div>
                   </div>
                 </div>
-                <div>
+                <div class="row row-lr-margin">
+                  <div class="align-left col-sm-10 col-lr-pad">
                   <p class="fs-14 w-400">Select submission visibility:</p>
```

Comment:
@Alabhya268 Can you please indent the content of the `<div>` tag.

Reply:
> @Alabhya268 minor change

@Ram81, I have fixed the indent, Please look at this PR again.